### ISSUE TITLE
Password entry - Machine when pasting the password into the input field

### ISSUE THREAD

**Issue by rleitan** [mozilla-iam/auth0-custom-lock#241]
[Steps to reproduce]:
1.Open sso.allizom.org
2.Enter a valid LDAP email address 
3.Go to the password field and paste the valid password ( using mouse or keyboard Ctrl+V)

[Expected result]:
Password entry - Human

[Actual result]:
Password entry - Machine

**Comment by hmitsch** [comment]
@gene1wood, as the requester of the password manager detection capability in NLX: Could you please take a look at this issue and Hidde's comment above. We would love to get your view.

-Henrik

**Comment by gene1wood** [comment]
Yes, that's correct. If a user copy pastes a password we want to consider it as a machine entry. This is because far more likely than the user manually typing their memorized password in to another window, then copying and pasting it into the browser is the user copy and pasting it from their password manager.
